ucsb-cs156-f24/proj-courses-f24-02
# Stryker tests for SectionsInstructorTable.js

Closes issue #6. Adds stryker tests to check that disableGroupBy is true for all columns.

To Test:
- First run npx stryker run -m src/main/components/Sections/SectionsInstructorTable.js in the frontend directory
- See that's mutation testing fails when //stryker disable comments are removed
- Then switch to this branch and see that test passes with no striker disable
